css/modal.css

```diff
@@ -0,0 +1,205 @@
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1000;
+  padding-top: 100px;
+  padding-bottom: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 70%;
+  border-radius: 10px;
+}
+
+/* The Close Button */
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
```

Comment:
- [ ] Kindly make sure you update the styling of your **Close Modal Button** to match the one from the Figma template

| Figma Design | Your Design |
| --------------------- | ----------------- |
| ![Screenshot from 2022-09-07 15-26-18](https://user-images.githubusercontent.com/83042742/188890277-70e9b2c5-a199-45c9-9d53-78037f188a45.png) | ![Screenshot from 2022-09-07 15-26-37](https://user-images.githubusercontent.com/83042742/188890325-45ced039-21a5-4fe4-877c-d7df85964dcc.png) |